app/payment/payment-complete.controller.test.js

```diff
@@ -142,7 +142,7 @@ describe('payment complete controller', () => {
       it('should redirect to the payment success page', () => {
         expect($('title').text()).to.include(`Your payment was successful - ${service.service_name.en}`)
         expect($('.govuk-header__content').text()).to.include(product.name)
-        expect($('#payment-reference').text()).to.include('ABC D123 4EF')
+        expect($('#payment-reference').text()).to.include('ABCD1234EF')
```

Comment:
is `include` appropriate here? We want to be sure that this exact value is present